pkg/storage/fs/ocis/ocis_unix.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018-2021 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// +build !windows
+
+package ocis
+
+import "syscall"
+
+func (fs *ocisfs) getQuota(path string) (uint64, uint64, error) {
+	stat := syscall.Statfs_t{}
+	err := syscall.Statfs(path, &stat)
+	if err != nil {
+		return 0, 0, err
+	}
+	total := stat.Blocks * uint64(stat.Bsize)                // Total data blocks in filesystem
```

Comment:
The calculation for `total` is different from the previous one
`total := ri.Size + (stat.Bavail * uint64(stat.Bsize)) // used treesize + available space`

Reply:
Oh damn you're right.
I copied this one from localfs...

Reply:
> total := ri.Size + (stat.Bavail * uint64(stat.Bsize)) // used treesize + available space

@butonic, the comment says `used treesize` but the `used` value is being calculated like this:
`used := (stat.Blocks - stat.Bavail) * uint64(stat.Bsize)`.
Is it intended to only use the available size?

Reply:
Oh nevermind, I read the comment wrong...